packages/common.nix

```diff
@@ -45,20 +45,51 @@
             }}
 
             function setup {
-                ${lib.getExe wineboot} --update
-                ${lib.getExe wine} msiexec /i "${wineUnwrapped}/share/wine/mono/wine-mono-9.3.0-x86.msi"
+                local revision="$1"
+                if [[ "$revision" -le 2 ]]; then
+                    echo "affinity-nix: Initializing wine prefix with mono, vulkan renderer and WinMetadata"
 
-                ${lib.getExe winetricks} renderer=vulkan
+                    ${lib.getExe wineboot} --update
+                    ${lib.getExe wine} msiexec /i "${wineUnwrapped}/share/wine/mono/wine-mono-9.3.0-x86.msi"
+
+                    ${lib.getExe winetricks} renderer=vulkan
+
+                    install -D -t "${affinityPath}/drive_c/windows/system32/WinMetadata/" ${dependencies}/*.winmd
+                fi
+                if [[ "$revision" -le 3 ]]; then
+                   echo "affinity-nix: Installing Microsoft WebView2 Runtime"
+
+                   ${lib.getExe wine} winecfg -v win7
+                   ${lib.getExe wine} "${dependencies}/MicrosoftEdgeWebView2RuntimeInstallerX64.exe" /silent /install
```

Comment:
The wine prefix could have its global version stuck on win7 if the webview2 runtime install fails. But before the affinity installer gets called the global version gets set to win11, so this is probably fine.